packages/notifi-react-example/src/NotifiCard/NotifiCard.css

```diff
@@ -2,3 +2,23 @@
   width: 400px;
   position: relative;
 }
+
+.NotifiInput__icon {
+  display: none;
+}
+
+.NotifiAlertListContainer__container{
+  display: none;
+}
+
+.NotifiAlertList__listItem{
+  display: none
+}
+
+.NotifiPreviewCard__backArrowContainer{
+  display: none;
+}
+
+/* .NotifiPreviewCard__editButton{
+  display: none;
+} */
```

Comment:
oh we probably don't want this committed to react example

Reply:
removed!